package.json

```
@@ -73,7 +73,7 @@
     "changelog": "conventional-changelog -p angular -i CHANGELOG.md -s",
     "release": "node scripts/release.js",
     "docs": "run-p dev docs-dev",
-    "docs-dev": "node ./bin/vitepress dev docs",
+    "docs-dev": "wait-on ./dist/node/cli.js && node ./bin/vitepress dev docs",
```

```suggestion
    "docs-dev": "wait-on -d 1000 dist/node/cli.js && node ./bin/vitepress dev docs",
```

Add some delay here to avoid race conditions as deleting dist might take some time in certain cases.